src/edgepi/calibration/edgepi_eeprom.py

```diff
@@ -306,3 +328,25 @@ def init_memory(self):
             self.log.info(f'{mem_size}bytes of data is read from the user space')
 
         return is_full, is_empty
+
+    def eeprom_reset(self):
+        """
+        Reset User space memory 
+        Args:
+            N/A
+        Return:
+            N/A
+        """
+        self.log.info("EEPROM Reset: User Space Memory Reset")
+
+        start_address_page = EdgePiMemoryInfo.USER_SPACE_START_BYTE.value
+        reset_vals = [255]*(EdgePiMemoryInfo.USER_SPACE_END_BYTE.value -\
+                            start_address_page + 1)
+        page_n = self.__generate_list_of_pages(start_address_page, reset_vals)
+
+        mem_offset = start_address_page
+        for page in page_n:
+            self.__page_write_register(mem_offset, page)
+            mem_offset = mem_offset+len(page)
+            time.sleep(0.002)
```

Comment:
not a big deal but I think wen rewrite this to be more memory efficient.

if I understand this code correctly, it is going to create a list of pages that all have the same value 255 for all the bytes.

since you know the page size, wouldn't that make more sense to create only one page of 255s and keep writing that?

there might be some edge case though at first and last pages maybe?